builders/go116/BUILD.bazel

```diff
@@ -15,7 +15,7 @@ builder(
             "//cmd/go/of_functions_framework:of_functions_framework.tgz",
         ],
     },
-    image = "openfunction/builder-go:v2",
+    image = "openfunction/builder-go:v2.3.0",
```

Comment:
v2.3.0-1.16?

Reply:
fixed. by default we should create a `v2.3.0` as well by `docker tag openfunction/builder-go:v2.3.0-1.16 openfunction/builder-go:v2.3.0`